tests/evolution/test_diversity_configuration.py

```
@@ -0,0 +1,73 @@
+"""
+Test suite for diversity configuration in the evolution system.
+
+This module tests that the evolution system correctly configures diversity
+calculators based on the EvolutionConfig.diversity_method setting.
+"""
+
+import pytest
+from unittest.mock import AsyncMock, MagicMock
```

Ruff (0.12.2)

`unittest.mock.AsyncMock` imported but unused

Remove unused import: `unittest.mock.AsyncMock`

(F401)